packages/components/src/icon-button/index.js

```diff
@@ -56,7 +55,7 @@ function IconButton( props, ref ) {
 			className={ classes }
 			ref={ ref }
 		>
-			<Icon icon={ icon } ariaPressed={ ariaPressed } />
+			<Icon icon={ icon } />
```

Comment:
Are you sure that it's safe to stop passing down this prop?

Reply:
Yes, it's basically being replaced by this line here:
https://github.com/WordPress/gutenberg/blob/82a96dd640354ba5068dee5a10e6195e96ea65bb/packages/components/src/button/index.native.js#L97
Were we send the `colorScheme` and the `active` state to all children of Button.

But considering @Tug's comment here: https://github.com/WordPress/gutenberg/pull/17356/files#r328582023 I might need to change this anyway.

What do you think?

Reply:
I checked the history and it looks like `ariaPressed` was introduced in https://github.com/WordPress/gutenberg/pull/11827. It was never really used in the web code. Should it be also removed in other places?

Reply:
Filed here: https://github.com/WordPress/gutenberg/pull/17741.